source/docs/software/getting-started/index.rst

```diff
@@ -12,7 +12,6 @@ Getting Started
    macos-setup
    linux-setup
    3rd-party-libraries
-   imaging-your-roborio
```

Comment:
I think we need this here.  This is an important getting started step.

Reply:
The Getting Started section should really be renamed to WPILib getting started, as it is much more pertinent to WPILib setup. This enforces the per topic structure.

Reply:
This section had everything that programmers need to do at a Robot Quick Build session. If you move imaging out, it breaks that. 

If you wanted it to be just WPILib getting started, there's other things that should be moved out as well, like radio programing and updating the PCM and PDP.

Reply:
Yes, those should be moved out as well. The idea was to create an actual getting started article directing users on systems. Even on ScreenSteps, the getting started article was never a "1 to boom robot".

Reply:
What was missing?

Reply:
This was discussed in a WPILib meeting awhile ago on the reasoning of the per-topic structure. Getting-Started may have been a "how-to" guide for rookie teams but caused numerous other problems. Every single article in getting-started was duplicated in various parts of the documentation (not linked to a single page but duplicated so inaccuracies between them). There was also the issue of coming to ScreenSteps looking for specific things and being unable to find them due to the poor organization and the "how-to" structure. If you come to frc-docs wanting to know how to use shuffleboard, you shouldn't have to dig through 50+ pages to find a single article. Per-topic alleviates most of the problems with very little cost. The trade-off that was decided was to create an actual introduction article to the FRC environment *without* cluttering up the docs. 

The reasons outlined above are why this pull-request has not been merged yet. I simply haven't had the time to create the article with the content referencing things in the various places.

Reply:
I'm ok with that, but don't think this should move until the other parts are moved in a separate PR.